docs/pages/appendices.md

```diff
@@ -0,0 +1,303 @@
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>sequencerFolderPath</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>sequencerName</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>sequencerPosition</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>containerSerialNumber</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>containerModel</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>laneCount</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>healthType</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>startDate</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>completedDate</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>pairedEndRun</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>software</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>metrics</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>poolNames</td>
+		<td>✓</td>
+		<td></td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>runBasesMask</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>bclCount</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>callCycle</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>chemistry</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>imgCycle</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>indexLengths</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>numCycles</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>numReads</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>poolNames</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>readLength</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>scoreCycle</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>workflowType</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>runType</td>
+		<td></td>
+		<td></td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>protocolVersion</td>
+		<td></td>
+		<td></td>
+		<td>✓</td>
+	</tr>
+</table>
+
+
+
+
+<a id="C" />
+### Appendix C: Sequencers Supported by Run Scanner
+This is a list of sequencers for which Run Scanner is capable of processing 
+output. 
+**Bold** items are verified to work with Run Scanner. 
+_Italic_ items are under development. Other items should work through having 
+equivalent output, however they have not been tested. 
+If a sequencer is not included in this list, it should be assumed to be 
+unsupported.
+
+#### PacBio
+* **RS**
+* **RS II**
+* _Sequel - Under development_
+
+#### Illumina
+* **HiSeq 1000**
+* **HiSeq 2000**
+* **HiSeq 2500**
+* **HiSeq X**
+* **MiSeq**
+* **NextSeq 500**
+* **NextSeq 550**
+* **NovaSeq 6000**
+* iSeq
+* MiniSeq
+
+#### Oxford Nanopore
+
+* **PromethION** - as of Release 19.01.1
+* _MinION - as of Release 18.12_
```

Comment:
Could it be clarified what software the Release numbers refer to? Right now it could be read as being a run scanner release version!

Reply:
Updated